### PR TITLE
update helm charts for latest PRs

### PR DESCRIPTION
**Pull request by MaxFedotov**
Hi @bsctl,
I've added params from my latest Capsule commits to helm charts and tested them on our test cluster - everything working great, so waiting for you to publish them to master branch

### CODE REVIEW

File: templates/deployment.yaml [hunk 1]
```diff
@@ -51,6 +51,8 @@ spec:
           - --enable-leader-election
           - --zap-log-level={{ default 4 .Values.log_level }}
           - --force-tenant-prefix={{ default false .Values.force_tenant_prefix }}
+          - --protected-namespace-regex={{ default "" .Values.protected_namespace_regex }}
```
Comment (prometherion) [inline templates/deployment.yaml:54]: Should these parameters follow camel case convention?

Comment (MaxFedotov) [reply templates/deployment.yaml:54]: They should, but you can't use `-` in variable names in helm :(

**Review by bsctl — COMMENTED** [review]
@prometherion is the `capsule-system` namespace hard coded? Can we leave Helm to use the `{{ .Release.Namespace }}` instead ?

**Review by prometherion — APPROVED** [review]
LGTM

**Comment by prometherion** [comment]
> @prometherion is the capsule-system namespace hard coded? Can we leave Helm to use the {{ .Release.Namespace }} instead ?

Oh, missed this: actually, the namespace is hard-coded but I guess we should fix this, so please, don't change the Helm Chart and let's open an Issue on Capsule rather to make it configurable.

**Pull request merged**